lib/pf/StatsD.pm

```diff
@@ -38,6 +39,7 @@ sub new {
     $host = 'localhost' unless defined $host;
     $port = 8125        unless defined $port;
     my $hostname = hostname; 
+    $hostname =~ s/\./_/g; # replace dots with underscores 
```

Comment:
Because it's used as the statsd delimiter already.
I'll make this more obvious.